terraform/registry/bucket.tf

```diff
@@ -0,0 +1,36 @@
+resource "aws_s3_bucket" "sunodo_bucket" {
+  bucket = "${var.bucket_name}"
+  force_destroy = false
+
+  tags = {
+    Name        = "sunodo_bucket"
+    Environment = "Dev"
+  }
+}
+
+resource "aws_s3_bucket_acl" "sunodo_bucket_acl" {
+  bucket = aws_s3_bucket.sunodo_bucket.id
+  acl    = "private"
+}
+
+resource "aws_s3_bucket_public_access_block" "sunodo_bucket_block_pub_access" {
+    bucket = aws_s3_bucket.sunodo_bucket.id
+    block_public_acls   = true
+    block_public_policy = true
+    restrict_public_buckets = true
+ }
+
+resource "aws_s3_bucket" "sunodo_log_bucket" {
```

Comment:
Who is gonna use this `sunodo_log_bucket`?

Let's keep the scope of this PR just for the sunodo-registry S3 Bucket

Reply:
Fixed !